### PR TITLE
Bug fixes for hosting in sagemaker-experiments example.

### PR DESCRIPTION
**Pull request by KenHenderson**
*Issue #:* https://github.com/awslabs/amazon-sagemaker-examples/issues/1063

*Description of changes:* Minor updates to reference running experiment example in Studio; fixed endpoint hosting but in custom Model in experiment python script.

Changes were validated with an end to end execution in Amazon SageMaker Studio in us-east-2 of the notebook and associated python script.  See the following PDF (export to HTML; print to PDF) [mnist-handwritten-digits-classification-experiment.pdf](https://github.com/awslabs/amazon-sagemaker-examples/files/4372227/mnist-handwritten-digits-classification-experiment.pdf) for the output of the notebook.  Account number was redacted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb [hunk 1]
```diff
@@ -449,20 +437,103 @@
...
    "outputs": [],
    "source": [
     "lineage_table.dataframe()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy endpoint for the best training-job / trial component\n",
```
Comment (danabens) [inline sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb]: seems unrelated to experiments. link to a deploy related notebook or documentation.

Comment (KenHenderson) [reply sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb]: Fair comment; wiki doc example that has some issues shows going from a TC to deployed model.  I opened issues with the relevant parties to correct the Studio experience and the doc for the example.  In the mean time this keeps that same example flow which seemed reasonable.  (Should be stable over time as well.)

File: sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb [hunk 2]
```diff
@@ -449,20 +437,103 @@
...
+    "    initial_instance_count=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cleanup\n",
+    "\n",
+    "Once we're doing don't forget to clean up the endpoint to prevent unnecessary billing.\n",
```
Comment (wangweixun) [inline sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb]: Please be explicit that Experiments does not cost you.

Comment (KenHenderson) [reply sagemaker-experiments/mnist-handwritten-digits-classification-experiment.ipynb]: Message calls out "endpoint" already.
